certified-connectors/Meisterplan/apiDefinition.swagger.json

```diff
@@ -1501,7 +1546,396 @@
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "409": {
+            "description": "Conflict"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          }
+        }
+      }
+    },
+    "/v1/resources/{resourceId}/absences": {
+      "get": {
+        "tags": [
+          "Absences"
+        ],
+        "summary": "Get a List of Absences",
+        "description": "Returns a list of absences of the resource.",
+        "operationId": "GetAllAbsences",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "finishDate",
+            "in": "query",
+            "description": "Inclusive finish date for filtering time period.",
+            "required": false,
+            "type": "string",
+            "format": "date",
+            "x-example": "2019-12-31",
+            "x-ms-summary": "FinishDate"
+          },
+          {
+            "name": "resourceId",
+            "in": "path",
+            "description": "Internal Meisterplan identifier",
+            "required": true,
+            "type": "string",
+            "x-example": "UUID",
+            "x-ms-summary": "ResourceId",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "startDate",
+            "in": "query",
+            "description": "Start date for filtering time period",
+            "required": false,
+            "type": "string",
+            "format": "date",
+            "x-example": "2019-01-31",
+            "x-ms-summary": "StartDate"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ListResponseOfAbsenceResponse"
+            }
+          },
+          "400": {
+            "description": "Bad Request"
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "404": {
+            "description": "Not Found"
+          },
+          "429": {
+            "description": "Too Many Requests"
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Absences"
+        ],
+        "summary": "Create Absences",
+        "description": "Use this request to add an absence for a certain time period. This time period can be defined using the start and finish attributes of the request. Using startDayType or finishDayType, you can specify whether the first/last day of the absence is a full day or only half a day. If the time interval consists of only one day, entering half day in startDayType and/or finishDayType will result in an half day (i.e., if startDayType is half a day and finishDayType is a whole day, then the absence refers to a half day).",
```

Comment:
@itd-sk I would suggest you keep all operations' descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Reply:
@Amjed-Ayoub shortened description